spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -108,6 +109,9 @@ public void registerBeanDefinitions(AnnotationMetadata importingClassMetadata, B
         registry.registerBeanDefinition("commandHandlerSubscriber",
                                         genericBeanDefinition(CommandHandlerSubscriber.class).getBeanDefinition());
 
+        registry.registerBeanDefinition("queryHandlerSubscriber",
```

Comment:
Question: Will we also add the `QueryBus` to the Spring Boot Auto configuration set up, or do we regard that as a separate issue/task?